Importer/Interpreter/Field/DatetimeFieldInterpreter.php

```diff
@@ -38,4 +41,18 @@ public function interpret(FieldConfigurationInterface $configuration, $value)
 
         return $dateTime;
     }
+
+    /**
+     * @param string $value
+     *
+     * @return bool
+     */
+    private function containsTime($value)
+    {
+        if (preg_match("/([0-2]+[0-9]):([0-5]?[0-9]):([0-5]?[0-9])$/", $value)) {
```

Comment:
Well I think we do not have to handle the case that we do not have seconds.

I think the whole idea is if the users have not provided any time, to add by default the "00:00:00". I thought it makes sense to check this, but if not then I could just do:

`$dateTime = \DateTime::createFromFormat($configuration->getFormat(), $value . ' 00:00:00');`